website/docs/mutation.md

```diff
@@ -252,6 +252,22 @@ must start with `:` or `@`. Also, if `assignPath` is set to a value which could
 be interpreted as a domain, such as `my.repo.lib/app`, then `assignDomain` must
 also be specified.
 
+### Mutation Annotations
+
+You can have two recording annotations applied at mutation time by enabling the `--mutation-annotations` flag.
```

Comment:
Worth noting that enabling this flag technically breaks the idempotence requirement K8s has for mutation webhooks?

Reply:
I'm ok either way. If we were to add it, we can link to https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#idempotence and append it to https://open-policy-agent.github.io/gatekeeper/website/docs/customize-startup#beta-enable-mutation-logging-and-annotations

Reply:
Sorry about the delay, closing the loop on this one.  Are we ok to merge this one as is or would you prefer me to add the stipulation?

Reply:
+1 on adding it since you are adding the flag to the docs

Reply:
Updated